### DISCUSSION TITLE
Allowed-Amount File: How do we represent data from a claim service line with multiple modifiers on it?

### DISCUSSION THREAD

**Discussion by rakeshkatte** [CMSgov/price-transparency-guide#408] [Q&A] [answered]
When a service line has multiple modifiers on it, it won't be clear as to which modifier/s contributed and by how much to the pricing of the line. So, it would be incorrect to stamp the same allowed amount for each of the modifiers. We are thinking of only using the primary modifier to publish the allowed amounts. We know it would be inaccurate but there is no other good alternative either. 

Are there any solutions or ideas to handle this situation?

Thanks in advance.

**Comment by tdfow** [comment]
@rakeshkatte after the 20 claim count rule logic has been met, then identify the unique allowed amounts.  If there are unique allowed amounts where some claims had modifiers and some claims did not have modifiers then we plan to only populate the modifiers if the claim's unique allowed amount had modifiers that met business rules of impacting the allowed amount.  Since not all modifiers alter the price--this type of approach could result in scenarios with or without modifier data.  If the business rules identifies certain modifiers have no impact in the allowed amount, then those would not be included.
You can report multiple modifiers, as the schema is defined as an array.

**Comment by rakeshkatte** [comment]
@tdfow Thank you for your response. We have a scenario where the line has both the
assistant surgeon and bilateral surgery modifiers. And both will take a
percentage reduction. So, in this case we don't know how to report the
amounts.

Here is an example:
Procedure code 26593 could come with modifier 50 (Bilateral) and 80 (assistant surgeon).
So, if the allowed amount for 26593 was $1000, this line would be priced at $1800 (1000 + 500 + 300).
Because we won't have the split up, we are not sure how to report this.

**Comment by shaselton-usds** [comment, marked as answer]
@rakeshkatte Just making sure, but is your question about the specifics of how much the modifier (if there are multiple) contributed to allowed amount?

If that is the case, there is only a single value to be reported for the allowed amount even with multiple modifiers. That detailed information about which modifier contributed to the allowed amounts will be lost with this type of reporting. You will still need to include all of the modifiers that contributed to the allowed amount on the claim.

If you have thoughts on how to appropriately disclose what modifier contributed what amount, we love to hear it, but it won't be required for v1.0.
